themes/sensei-course-theme/single.php

```diff
@@ -45,7 +45,8 @@
 
 	<!-- wp:column {"width":"","className":"sensei-course-theme__main-content"} -->
 	<div class="wp-block-column sensei-course-theme__main-content">
-		<!-- wp:post-title /-->
+		<!-- wp:sensei-lms/course-theme-module-title /-->
```

Comment:
> At the moment, the default tags for post title is h1 and for module title is h2. Does that work?

🤔 I think it would be interesting to use something else here. The headings are part of the same scope of headings, so it would mean that the module (`h2`) would be one level below (`h1`) lesson, which is not exactly true. I see the module name more as a "identifier", like a type of content, than a title. Does it make sense?